src/rabbit_auth_backend_ldap.erl

```diff
@@ -504,6 +507,56 @@ creds(#auth_user{impl = #impl{user_dn = UserDN, password = PW}}, as_user) ->
 creds(_, Creds) ->
     {ok, Creds}.
 
+%% Maybe scrub credentials
+maybe_scrub_creds([], Acc)      -> lists:reverse(Acc);
+maybe_scrub_creds([H|Rem], Acc) -> maybe_scrub_creds(Rem, [maybe_scrub_cred(H)|Acc]).
+
+maybe_scrub_cred({'BindRequest', N, DN, {simple, PWD}}) ->
```

Comment:
If it scrubs only `BindRequest`, should setting be called `scrub_bind_request_credentials`?
Because it doesn't seem like not logging bind credentials.
`LDAP bind succeeded: cn=Alice,ou=People,dc=example,dc=com` can look like logging credentials for user.